org.uqbar.project.wollok.lib/src/wollok/game.wlk

```diff
@@ -1,118 +1,123 @@
-class Key {	
-	var keyCodes
-	
-	constructor(_keyCodes) {
-		keyCodes = _keyCodes
-	}
+object keyboard {
 
-	/**
-	 * Adds a block that will be executed always self is pressed.
-	 */	
-	method onPressDo(action) {
-		keyCodes.forEach{ key => game.whenKeyPressedDo(key, action) } //TODO: Implement native
-	}
-}
+	method any() = new Key(-1)
 
-object ANY_KEY inherits Key([-1]) { }
+	method num_0() = new Key(7, 144)
```

Comment:
What if we remove the _, there are no _s in the Wollok code base, I think.

Reply:
Done!